pkg/sql/crdb_internal.go

```diff
@@ -736,7 +768,8 @@ func (p *planner) makeSessionsRequest(ctx context.Context) serverpb.ListSessions
 // crdbInternalLocalQueriesTable exposes the list of running queries
 // on the current node. The results are dependent on the current user.
 var crdbInternalLocalQueriesTable = virtualSchemaTable{
-	schema: fmt.Sprintf(queriesSchemaPattern, "node_queries"),
+	schema:  fmt.Sprintf(queriesSchemaPattern, "node_queries"),
+	comment: "the list of running queries on the current node. The results are dependent on the current user.",
```

Comment:
inconsistent articles (either use "the" or not at start, but do the same consistently)